### PR TITLE
:sparkles: Add `kubeshark-worker-metrics` service and document it

### PR DESCRIPTION
**Pull request by corest**
Add metrics k8s service and documentation sample on metrics scrape config configuration.
Depends on https://github.com/kubeshark/worker/pull/34

### CODE REVIEW

File: config/configStructs/tapConfig.go [hunk 1]
```diff
@@ -33,6 +33,8 @@ const (
 	DebugLabel                   = "debug"
 	ContainerPort                = 80
 	ContainerPortStr             = "80"
+	MetricsPort                  = 49100
+	MetricsPortStr               = "49100"
```
Comment (mertyildiran) [inline config/configStructs/tapConfig.go:37]:
```suggestion
```

Unused.

File: helm-chart/README.md [hunk 2]
```diff
@@ -168,6 +172,7 @@ helm install kubeshark kubeshark/kubeshark \
 | `scripting.env`                           | Environment variables for the scripting      | `{}`                                                    |
 | `scripting.source`                        | Source directory of the scripts                | `""`                                                    |
 | `scripting.watchScripts`                  | Enable watch mode for the scripts in source directory          | `true`                                                  |
+| `prometheus.service.port`                  | Pod port used to expose Prometheus metrics          | `49100`                                                  |
```
Comment (mertyildiran) [inline helm-chart/README.md:175]:
```suggestion
| `tap.metrics.port`                  | Pod port used to expose Prometheus metrics          | `49100`                                                  |
```

File: helm-chart/templates/16-worker-service-metrics.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    prometheus.io/port: '{{ .Values.tap.metrics.port }}'
+spec:
+  selector:
+    app.kubeshark.co/app: worker
+    {{- include "kubeshark.labels" . | nindent 4 }}
+  ports:
+  - name: metrics
+    protocol: TCP
+    port: {{ .Values.tap.metrics.port }}
+    targetPort: {{ .Values.tap.metrics.port }}
```
Comment (mertyildiran) [inline helm-chart/templates/16-worker-service-metrics.yaml:18]:
```suggestion
    targetPort: {{ .Values.tap.metrics.port }}

```

Newline.

**Review by mertyildiran — CHANGES_REQUESTED** [review]

**Comment by corest** [comment]
@mertyildiran thx for the review. Committed all the suggestions

**Review by mertyildiran — APPROVED** [review]

**Pull request merged**